sdk/core/azure-core-test/src/main/java/com/azure/core/test/utils/TestProxyUtils.java

```diff
@@ -53,16 +57,16 @@ public class TestProxyUtils {
         "primaryKey", "secondaryKey", "adminPassword.value", "administratorLoginPassword", "runAsPassword",
         "adminPassword", "accessSAS", "WEBSITE_AUTH_ENCRYPTION_KEY", "decryptionKey", "primaryMasterKey",
         "primaryReadonlyMasterKey", "secondaryMasterKey", "secondaryReadonlyMasterKey", "certificatePassword",
-        "clientSecret", "keyVaultClientSecret", "authHeader", "httpHeader", "encryptedCredential", "appkey",
-        "functionKey", "atlasKafkaPrimaryEndpoint", "atlasKafkaSecondaryEndpoint", "certificatePassword",
-        "storageAccountPrimaryKey", "privateKey", "fencingClientPassword", "acrToken", "scriptUrlSasToken",
-        "azureBlobSource.containerUrl", "properties.DOCKER_REGISTRY_SEVER_PASSWORD");
+        "clientSecret", "keyVaultClientSecret", "authHeader", "httpHeader", "encryptedCredential", "functionKey",
+        "atlasKafkaPrimaryEndpoint", "atlasKafkaSecondaryEndpoint", "certificatePassword", "storageAccountPrimaryKey",
+        "privateKey", "fencingClientPassword", "acrToken", "scriptUrlSasToken", "azureBlobSource.containerUrl",
+        "properties.DOCKER_REGISTRY_SEVER_PASSWORD");
 
     private static final List<TestProxySanitizer> HEADER_KEY_REGEX_TO_REDACT = Arrays.asList(
 
         // upper and lower case versions intentional
-        new TestProxySanitizer("Operation-location", URL_REGEX, REDACTED_VALUE, TestProxySanitizerType.HEADER),
-        new TestProxySanitizer("Operation-Location", URL_REGEX, REDACTED_VALUE, TestProxySanitizerType.HEADER),
+        // new TestProxySanitizer("Operation-location", URL_REGEX, REDACTED_VALUE, TestProxySanitizerType.HEADER),
```

Comment:
should these be left commented out ?
If so, maybe we should add a comment explaining why.

Reply:
we can simply remove these. They are added to the central sanitizer list.